pkg/apis/core/types.go

```diff
@@ -488,6 +496,18 @@ type PersistentVolumeClaimSpec struct {
 	// (Alpha) Using the namespace field of dataSourceRef requires the CrossNamespaceVolumeDataSource feature gate to be enabled.
 	// +optional
 	DataSourceRef *TypedObjectReference
+	// volumeAttributesClassName may be used to set the VolumeAttributesClass used by this claim.
+	// If specified, the CSI driver will create or update the volume with the attributes defined
+	// in the corresponding VolumeAttributesClass. This has a different purpose than storageClassName,
+	// it can be changed after the claim is created. An empty string value means that no VolumeAttributesClass
+	// will be applied to the claim but reset it to empty string is not allowed once it is set.
+	// If unspecified and the PVC is unbound, the default VolumeAttributesClass will be set by the
+	// persistentvolume controller if it exists.
```

Comment:
s/PVC/PersistentVolumeClaim/ ?

What happens if the PVC is bound?

Reply:
We support modifyVolume when the volume is in use. The modify volume operation needs to be non-disruptive.